kyuubi-server/src/main/scala/org/apache/kyuubi/engine/KyuubiApplicationManager.scala

```diff
@@ -211,4 +213,9 @@ object KyuubiApplicationManager {
       case _ =>
     }
   }
+
+  def sessionUploadFolderPath(sessionId: String): Path = {
+    require(StringUtils.isNotBlank(sessionId))
+    uploadWorkDir.resolve(sessionId)
+  }
```

Comment:
how about moving to `KyuubiApplicationManager.scala`? @bowenliang123

Reply:
SGTM.